index.js

```diff
@@ -7,12 +7,15 @@ var extend = require('deep-extend');
 var assets = {};
 var DEFAULT_OUTPUT_FILENAME = 'webpack-stats.json';
 var DEFAULT_LOG_TIME = false;
+var DEFAULT_ASSET_CHUNK_NAME = 'assets';
 
 
 function Plugin(options) {
   this.contents = {};
   this.options = options || {};
   this.options.filename = this.options.filename || DEFAULT_OUTPUT_FILENAME;
+  this.options.trackAssets = this.options.trackAssets || false;
```

Comment:
trackAssets is off by default so there will be no change from default behavior of the app for those upgrading